packages/firestore/src/core/view.ts

```diff
@@ -280,13 +280,15 @@ export class View {
    *        change.
    * @param targetChange A target change to apply for computing limbo docs and
    *        sync state.
+   * @param fromBundle Whether the changes are from applying a bundle file.
    * @return A new ViewChange with the given docs, changes, and sync state.
    */
   // PORTING NOTE: The iOS/Android clients always compute limbo document changes.
   applyChanges(
     docChanges: ViewDocumentChanges,
     updateLimboDocuments: boolean,
-    targetChange?: TargetChange
+    targetChange?: TargetChange,
+    fromBundle: boolean = false
```

Comment:
It might make sense to move this to `ViewDocumentChanges`, which already has similar state (`needsRefill`).